### PR TITLE
Added a section for empty return types under naming conventions

### PR DESCRIPTION
**Pull request by htinlinn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -159,6 +160,20 @@ x.subViews(color: y)
 x.nounCapitalize()
 ```
 
+#### Empty Return Types ####
+
+When specifying return type for functions, methods or closures that return no value, favor the type alias `Void` over empty tuple `()`.
+
+**Preferred:**
+```swift
+let onTextEntry: (String) -> Void
```
Comment (ThibaultKlein) [inline README.md]: wondering if a function with a closure would be a better (at least more common) example

Comment (htinlinn) [reply README.md]: updated

File: README.md [hunk 2]
```diff
@@ -159,6 +160,20 @@ x.subViews(color: y)
 x.nounCapitalize()
 ```
 
+#### Empty Return Types ####
+
+When specifying return type for functions, methods or closures that return no value, favor the type alias `Void` over empty tuple `()`.
+
+**Preferred:**
+```swift
+func performTask(_ completion: (Bool) -> Void)
```
Comment (ThibaultKlein) [inline README.md]: Shouldn't it be `func performTask(@escaping completion: (Bool) -> Void)`?

Comment (phillfarrugia) [reply README.md]: +1

**Review by phillfarrugia — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (htinlinn) [reply README.md]: Updated

**Review by ThibaultKlein — APPROVED** [review]

**Review by paulmiard — APPROVED** [review]

**Review by paulmiard — APPROVED** [review]

**Pull request merged**